docs/conceptual/core/consensus.md

```diff
@@ -2,14 +2,104 @@
 title: Consensus
 ---
 
-* What is consensus for on a blockchain?
+## State Machines and Conflicts
+A blockchain runtime is a [state machine](https://en.wikipedia.org/wiki/Finite-state_machine). It has some internal state, and state transition function that allows it to transition from its current state to a future state. In most runtimes there are states that have valid transitions to multiple future states, but a single transition must be selected. Consider the following examples.
 
-* Block Production
+### Voting Conflict
+A blockchain runtime tallys simple two-option votes for a  DAO. When a new election is opened, Alice, a member of the DAO, can vote Aye. Alice could also vote Nay. But Alice cannot vote both Aye and Nay in the same election.
 
-* Finality
+### Land Registry Conflict
+Consider a blockchain runtime that serves as a land registry for a futuristic nation-state. The registry keeps track of transfers of land ownership among citizens, and occasionally opens new plots of land to be claimed. For a particular newly opened land run, Alice could claim the plot, or Bob could claim it. But they cannot both claim it.
 
-* NPoS
+### Classic Double Spend Conflict
+A pure cryptocurrency, like Bitcoin, tracks the minting and spending of tokens. Alice holds a particular coin, so she can pay it to Bob. She could also pay that coin to Carol. But she cannot pay the same coin to both Bob and Carol.
 
-* PoW
+The unifying property in these examples is that there are multiple transactions, each of which is individually valid, but are mutually exclusive. Among the alternatives, a single transaction must be chosen.
 
-* What would it take to write my own consensus?
+
+## Conflict Exclusion
+In centralized systems, the central authority chooses among mutually exclusive alternatives by recording transactions in the order it sees them, and choosing the first of the competing alternatives when a conflict arises. In decentralized systems, the nodes will see transactions in different orders, and thus they must use a more elaborate method to exclude transactions. As a further complication, blockchain networks strive to be fault tolerant, which means that they should continue to provide consistent data even if up to one third of participants are not following the rules.
```

Comment:
`one third` rule only applied to BFT, but it may not a good idea to detail such principle, instead we can explicitly mention Substrate default consensus (GRANDPA) here.

Reply:
I've removed _this_ occurrence of the specific threshold, saving that conversation for later. When discussing GRANDPA I mentioned it again, and in the learn more section I link some resources that explain it.